deploy/deploy.bash

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env bash
+
+# commands run to deploy serverless app
+# inside the docker container
+# once validated to work, this script should be the CMD arg of the Dockerfile when deploying
+
+# the Dockerfile already set up environment variables in the container so that AWS configure can use them
+# they are put in /opt/aws
+# this is kinda cheezy
+echo "Configuring AWS credentials"
+export AWS_ACCESS_KEY_ID=$(awk 'NR == 2' /opt/aws | cut -f3 -d ' ')
+export AWS_SECRET_ACCESS_KEY=$(awk 'NR == 3' /opt/aws | cut -f3 -d ' ')
```

Comment:
This solution will not work as we can't have credentials baked in to the image. They have to be mounted by container from the host when new image starts!